examples/searchlight/example_searchlight.py

```diff
@@ -90,10 +93,36 @@ def sfn(l, msk, myrad, bcast_var):
   import matplotlib.pyplot as plt
   for (cnt, img) in enumerate(global_outputs):
     plt.imshow(img,cmap='hot',vmin=0,vmax=1)
-    plt.savefig('img' + str(cnt) + '.png')
+    plt.savefig('cube_img' + str(cnt) + '.png')
     plt.clf()
 
+# Searchlight using diamond-shaped searchlight region
+
+# Create searchlight object
+sl = Searchlight(sl_rad=2, max_blk_edge=5)
```

Comment:
I assume that `Diamond5x5x5` only works for masks with radius >= 2. Is it correct? If yes, please note it somewhere.